components/match2/wikis/chess/chess_eco.lua

```diff
@@ -0,0 +1,32 @@
+---
+-- @Liquipedia
+-- wiki=chess
+-- page=Module:Chess/ECO
```

Comment:
this does not fit the file name here
the automated checks etc pp will read this as `chess_e_c_o.lua`